pkgs/top-level/all-packages.nix

```diff
@@ -623,6 +623,7 @@ in
 
   mkShell = callPackage ../build-support/mkshell { };
   mkShellNoCC = mkShell.override { stdenv = stdenvNoCC; };
+  minimalMkShell = callPackage ../build-support/mkshell/minimal.nix { };
```

Comment:
We already have `python3Minimal` and `systemdMinimal`. Maybe it should than be `mkShellMinimal` for consistency?

Reply:
Will rename.